Assets/Scripts/Interactables/FallsAndBreaks.cs

```diff
@@ -0,0 +1,107 @@
+using System.Collections;
+using UnityEngine;
+
+[RequireComponent(typeof(MeshRenderer))]
+[RequireComponent(typeof(ParticleSystem))]
+[RequireComponent(typeof(BoxCollider))]
+[RequireComponent(typeof(Rigidbody))]
+public class FallsAndBreaks : Interactable
+{
+  private Transform player;
+  private new Rigidbody rigidbody;
+  private BoxCollider boxCollider;
+  private new ParticleSystem particleSystem;
+  private MeshRenderer meshRenderer;
+  private IEnumerator coroutine;
+  public float nudgeForce = 400f;
+
+  protected override void Start()
+  {
+    base.Start();
+    player = GameObject.FindGameObjectWithTag("Player").transform;
+    rigidbody = GetComponent<Rigidbody>();
+    boxCollider = GetComponent<BoxCollider>();
+    particleSystem = GetComponent<ParticleSystem>();
+    meshRenderer = GetComponent<MeshRenderer>();
+  }
+
+  // These three might be useful in other scripts for other interactables
+  void FixedUpdate()
+  {
+    isObjectMoving();
+  }
+
+  private void isObjectMoving()
+  {
+    // Use IsSleeping() to test if the object is moving
+    if (rigidbody != null && rigidbody.IsSleeping())
+    {
+      // If we run out of interactions, make the object essentially static
+      if (interactionCount == maxInteractionCount)
```

Comment:
why are we limiting the amount of interactions?

Reply:
@gabxgonz interactionCount is banished!